frontend/javascripts/oxalis/model/sagas/dataset_saga.ts

```diff
@@ -50,9 +51,13 @@ export function* watchZ1Downsampling(): Saga<void> {
     const minVoxelPerPixel = 0.1;
     if (!userClosedWarning) {
       // checking only the downsampled dimensions x and y
+      const is3DViewportFullscreen = yield* select(
```

Comment:
the `activeViewport` is the one that the mouse hovers currently (or most recently). it's not necessarily in fullscreen then. this means that the toast would likely disappear and re-appear when moving the mouse to the 3d viewport and out of it again.
to get the fullscreen info, I'd do something like the following:

```
const extent_xy = getViewportExtents(state)["PLANE_XY"];
const extent_yz = getViewportExtents(state)["PLANE_YZ"];
...
const areas = [extent_xy, extent_yz, ...].map(([width, height]) => width * height);
const areDataviewportsInvisible = _.sum(areas) == 0;
```

could be a helper function next to `getViewportExtents` in view_mode_accessors.

Reply:
@philippotto I applied your suggestion, please have a look again.